Makefile

```diff
@@ -539,6 +539,21 @@ endif
 unpack_dist:
 	@tar -xf $(THEMIS_DIST_FILENAME)
 
+nsis_installer: $(BIN_PATH)/InstallThemis.exe
```

Comment:
can we add a check that if this command will run on macOS, so users see error message?

Reply:
Sure, will do.

It will fail on macOS because `makensis` is unlikely to be installed there, but it's definitely better to have a human-readable error message for this case.